lib/puppet/provider/cisco_stp_global/cisco.rb

```diff
@@ -0,0 +1,298 @@
+# The NXAPI provider for cisco_stp_global
```

Comment:
@saichint A bit more background info.  The reason this change is needed is due to this now merged PR.  https://github.com/cisco/cisco-network-puppet-module/commit/efc8885cc8971b56514558d42b509de30d57bd47.  Since we support both nxapi and grpc, Cisco was a more appropriate name.